pad.c

```diff
@@ -712,7 +712,7 @@ Perl_pad_alloc(pTHX_ I32 optype, U32 tmptype)
         pad_reset();
     if (tmptype == SVs_PADMY) { /* Not & because this ‘flag’ is 0.  */
         /* For a my, simply push a null SV onto the end of PL_comppad. */
-        sv = *av_fetch(PL_comppad, AvFILLp(PL_comppad) + 1, TRUE);
+        sv = *av_store_simple(PL_comppad, AvFILLp(PL_comppad) + 1, newSV(0));
```

Comment:
Might be worth clarifying in the commit message that replacing a fetch with a store here really is intentional.

In any case, the first line of the commit message is rather longer than recommended; I suggest something more like:
```
Perl_pad_alloc: simplify av_fetch() calls

The nature of PL_comppad allows this.
```

Reply:
Thanks, I've changed it to your recommendation.

Reply:
Update pushed.